pkg/sql/schemachanger/scbuild/scbuildctx/errors.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package scbuildctx
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/errors"
+)
+
+type notImplementedError struct {
```

Comment:
I'd be down to move the errors out to either the root `schemachanger` package or an `scerrors` package. What do you think? Then, at that point, I think we could remove any imports of `scbuildctx`

Reply:
Great idea.